flocker/node/functional/test_docker.py

```diff
@@ -726,13 +728,15 @@ def test_command_line(self):
         name = random_name(self)
         d = self.start_container(
             name, image_name=u"busybox",
-            command_line=[u"sh", u"-c", u"""\
+            # Pass in pvector since this likely to be what caller actually
+            # passes in:
+            command_line=pvector([u"sh", u"-c", u"""\
 echo -n '#!/bin/sh
 echo -n "HTTP/1.1 200 OK\r\n\r\nhi"
 ' > /tmp/script.sh;
 chmod +x /tmp/script.sh;
 nc -ll -p 8080 -e /tmp/script.sh
-"""],
+"""]),
             ports=[PortMap(internal_port=8080,
                            external_port=external_port)])
 
```

Comment:
I copied some code from there, didn't intend to modify it. Bit confused how it worked before, or alternately how it works now.